### PR TITLE
fix(i18n,client): translatable cta url

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

Currently, a camper visiting the Chinese `/learn` page will see call to action links to help with the translation efforts. These links are hard-coded to point to the English version of the contributing documentation.

With this PR, those links are pulled out into the `translations.json` object so they can be replaced with a language-specific link to the appropriate internationalised version of our documentation.

### CODE REVIEW

**Comment by RandellDawson** [comment]
This seems more like a config than a translation, so I am not sure `translations.json` is the best place for this.  I would think we would want tighter control over these links than to rely on Crowdin integration to get these right.

**Comment by nhcarrigan** [comment]
That's a good point. 

Our footer links (for like the donate and about page) are also in the `translations.json` object. Should we extract those and compile a `links.json` object so we can control them through GitHub like we do the meta tags and trending news articles?

**Comment by RandellDawson** [comment]
> That's a good point.
> 
> Our footer links (for like the donate and about page) are also in the `translations.json` object. Should we extract those and compile a `links.json` object so we can control them through GitHub like we do the meta tags and trending news articles?

That is what I am thinking.

**Comment by nhcarrigan** [comment]
Will update the PR today when I have time

**Comment by nhcarrigan** [comment]
Okay @RandellDawson I've extracted the URL values from `translations.json` and created a new `links.json` to keep off of Crowdin.

File: client/i18n/locales/espanol/translations.json [hunk 1]
```diff
@@ -283,7 +283,7 @@
     "scrimba-tip": "Sugerencia: Si el mini-navegador cubre el código, haz clic y arrastra para moverlo. Además, siéntate libre de detener y editar el código en el video en cualquier momento.",
     "chal-preview": "Vista previa del desafío",
     "cert-map-estimates": {
-      "certs": "Certificación {{title}} (300 horas)",
+      "certs": "Certificación {{title}} (300 horas)",
```
Comment (nhcarrigan) [inline client/i18n/locales/espanol/translations.json:286]: Huh?? 👀

**Review by raisedadead — COMMENTED** [review]
Code LGTM. 

I see commits were pushed after the ready for QA was added, just confirming if this is good to go, before I pull it down.

**Comment by nhcarrigan** [comment]
This is good to go, yes

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
@freeCodeCamp/dev-team please approve.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
@nhcarrigan Great job on this PR!

**Pull request merged**